### PR TITLE
Adds missing endpoint + corrects others

### PR DESCRIPTION
**Pull request by gcaracuel**
Adds extra parameters + missing endpoints
Adds Unittest

### CODE REVIEW

**Comment by gcaracuel** [comment]
Note only changed name of internal functions that IMHO must not be exposed. If I really want to run a GET over the API I would do it without this wrapper.

**Comment by lagerfeuer** [comment]
oh sorry, that's a mistake on my end. internal functions are fine fine. you can leave that as is.

File: README.rst [hunk 1]
```diff
@@ -23,12 +23,27 @@ Import
 Methods
 =======
 
+Price
+-----
+.. code:: python
+
+   cryptocompare.get_price('BTC')
+   # or
+   cryptocompare.get_price('BTC',curr='USD',f ull=True)
```
Comment (lagerfeuer) [inline README.rst]: typo `f ull`

**Comment by lagerfeuer** [comment]
`curr` still needs to be changed to `currency`

File: test.py [hunk 2]
```diff
@@ -3,48 +3,192 @@
...
+
+    def test_get_price_given_exchange(self):
+        res = cryptocompare.get_price(coins[0:2], currencies=currencies[0:2], exchange='Kraken')
+        # If exchange parameters fails it will return an error so next assert won't be treu
+        self.assertIsNotNone(res[coins[0]])
+        self.assertIsNotNone(res[coins[1]])
+        self.assertIsNotNone(res[coins[0]][currencies[0]])
+        self.assertIsNotNone(res[coins[0]][currencies[1]])
+        self.assertIsNotNone(res[coins[1]][currencies[0]])
+        self.assertIsNotNone(res[coins[1]][currencies[1]])
```
Comment (lagerfeuer) [inline test.py]: missing:
```python
if __name__ == '__main__':
    unittest.main()
```

Comment (lagerfeuer) [reply test.py]: also, there are two errors when running your unittests: `NameError: name 'curr' is not defined`

**Pull request closed without merging**